src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -127,8 +127,8 @@ trait Formats
           "version" -> launched.appVersion
         )
       ){
-        case (launchedJs, ipAddresses) => launchedJs ++ Json.obj("ipAddresses" -> ipAddresses)
-      }
+          case (launchedJs, ipAddresses) => launchedJs ++ Json.obj("ipAddresses" -> ipAddresses)
```

Comment:
I think sbt formatted this file. But surely it's unrelated with my change and only introduce noise. Will revert this change